sql_commands.txt

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM players WHERE team = 'NYK';
+SELECT * FROM players WHERE age < 26;
+SELECT * FROM players ORDER BY points;
+SELECT name, points/games FROM players LIMIT 20;
```

Comment:
We need an `ORDER BY` here to ensure we're getting the list sorted by our top scores:
`SELECT name, points/games as points_per_game FROM players ORDER BY points_per_game DESC LIMIT 20;`